jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/AvatarTests.java

```diff
@@ -1,78 +1,55 @@
-        dataDisplayAvatarFrame.square.is().displayedImg();
-        dataDisplayAvatarFrame.square.is().hasImg("MuiAvatar-img");
-    }
-
-    @Test
-    public void roundedTest(){
-        dataDisplayRoundedAvatarPage.open();
-
-        dataDisplayAvatarFrame.rounded.is().displayed();
-        dataDisplayAvatarFrame.rounded.hasClass("MuiAvatar-rounded");
-        dataDisplayAvatarFrame.rounded.is().displayedImg();
-        dataDisplayAvatarFrame.rounded.is().hasImg("MuiAvatar-img");
-    }
 
-    @Test
-    public void fallbackAltTest(){
-        dataDisplayFallbackAltAvatarPage.open();
 
-        dataDisplayAvatarFrame.fallbackAlt.is().displayed();
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.fallbackAlt.is().text("L");
+public class AvatarTests extends TestsInit {
+    @BeforeTest
+    public void beforeTest() {
+        //openSection("Avatar");
+        StaticSite.dataDisplayAvatarPage.open();
+        StaticSite.dataDisplayAvatarPage.isOpened();
     }
 
     @Test
-    public void letterTest(){
-        dataDisplayLetterAvatarPage.open();
-
-        dataDisplayAvatarFrame.letter.is().displayed();
-        dataDisplayAvatarFrame.letter.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.fallbackAlt.is().text("A");
+    public void avatarTests() {
+        testLogic(avatarWithoutPhoto, true);
+        testLogic(avatarWithPhoto, false);
+        onlineStatus.get(1).has().classValue(Matchers.containsString("MuiBadge-dot"));
+        onlineStatus.get(2).has().text("R");
+        onlineStatus.get(2).has().classValue(Matchers.containsString("MuiBadge-anchorOriginBottomRightCircle"));
     }
 
-    @Test
-    public void iconTest(){
-        dataDisplayIconAvatarPage.open();
-
-        dataDisplayAvatarFrame.icon.is().displayed();
-        dataDisplayAvatarFrame.icon.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.icon.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.icon.is().displayedSvg();
-        dataDisplayAvatarFrame.icon.is().hasSvg("MuiSvgIcon-root");
+    private void testLogic(List<UIElement> elements, boolean noPhoto) {
+        for (int i = 1; i <= elements.size(); i++) {
+            elements.get(i).isDisplayed();
+            elements.get(i).hasImage();
+            if (i < 3) {
+                if (i == 1 && noPhoto)
+                    elements.get(i).has().text("L");
+                elements.get(i).has().classValue(Matchers.containsString("MuiAvatar-circle"));
+            }
+            if (i == 3) {
+                elements.get(i).has().text("A");
+                elements.get(i).has().classValue(Matchers.containsString("MuiAvatar-square"));
+                elements.get(i).has().classValue(Matchers.containsString("jss3"));
```

Comment:
we don't need to check this value because it's dynamic value and changes when you refresh page

Reply:
actually, jss3 and jss4 a persistent and contains color of the avatar.